spec/models/search_query_spec.rb

```diff
@@ -77,6 +77,123 @@
     end
   end
 
+  describe 'self.find_or_create' do
+    context 'search queries' do
+      let(:user) { create(:disclosure_bmt_user) }
+
+      let(:parent_search_query) {
+        create(
+          :search_query,
+          user:        user,
+          query_type:  'search',
+          search_text: 'courts',
+        )
+      }
+      let(:existing_search_query) {
+        create(
+          :search_query,
+          user:                   user,
+          query_type:             'search',
+          search_text:            'courts',
```

Comment:
Winnie-the-Pooh seems like a much better search term =)

Reply:
I can change that. :)

Reply:
I can change that.